src/pages/workspace/accounting/PolicyAccountingPage.tsx

```diff
@@ -129,7 +129,7 @@ function accountingIntegrationData(
                     />
                 ),
                 onImportPagePress: () => {},
-                onExportPagePress: () => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_NETSUITE_EXPORT.getRoute(policyID)),
```

Comment:
I think during one of the merges this navigate got messed up. The current main code redirects Sage Export link to NetSuite. Added a fix here.